jobs/ServiceInstanceUpdateJob.js

```diff
@@ -134,41 +146,49 @@ class ServiceInstanceUpdateJob extends BaseJob {
       });
   }
 
-  static getOutdatedDiff(deploymentName, instanceId, plan) {
+  static deploymentStaggered(err) {
```

Comment:
Why do we have to do this response description parsing logic? Does not look good.

Reply:
I was present in FabrikOperation class. I moved it here.